extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/README.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# FS Internet Radio Binding
+
+This binding integrates internet radios based on the [Frontier Silicon chipset](http://www.frontier-silicon.com/).
+
+![Medion Internet Radio](doc/Medion-Internet-Radio.png)
```

Comment:
Where did you get the two pictures from? Do you own the copyright and thus can contribute them under EPL?
If not, please remove them - or take a picture yourself and use it.